### PR TITLE
Use Jekyll theme

### PR DESCRIPTION
**Pull request by zzxwill**
- Use jekyll theme which is highly recommended by
official Github Pages.
- add contribution guide #2

### CODE REVIEW

**Comment by hongchaodeng** [comment]
fix https://github.com/oam-dev/kubevela.io/issues/2

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
 # KubeVela Website
+[kubevela.io](https://kubevela.io)
+
+## Contribution
```
Comment (hongchaodeng) [inline README.md:4]:
```suggestion
## Contributing
```

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+```
+
+- Make your changes
+
+If you would like to override the default css of Jekyll, please revise [./assets/main.scss](./assets/main.scss)
+
+- Build static files
+
+Generate static files into folder `docs` which will be the source for Github Pages.
+```
```
Comment (hongchaodeng) [inline README.md:20]:
```suggestion
The following command uses [bundler](https://bundler.io/).
```
```

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
 # KubeVela Website
+[kubevela.io](https://kubevela.io)
+
+## Contribution
+
+- Fork and clone
+
+```
```
Comment (hongchaodeng) [inline README.md:8]: Please pay attention to the indentation to all of the lines within points.

Comment (hongchaodeng) [reply README.md:8]: Do something like:

- Fork and clone

  ```
  $ git clone git@github.com:xxx/kubevela.io.git
  $ cd kubevela.io
  ```

**Review by wonderflow — APPROVED** [review]

**Pull request merged**